packagers/pnpm-workspaces-default-output/test.sh

```diff
@@ -12,7 +12,7 @@ if [ -f "$FILE1" ] || [ -f "$FILE2" ]; then
     echo "Client should not generate in sub-folder"; exit 1;
 fi;
 
-if [ ! -f "$FILE0" ]; then
+if [ ! -d "$FILE0" ]; then
```

Comment:
nitpick:
-f is for checking if the file exist and -d for a directory Ithink @millsp?
So it's not a file like `$FILE0` suggests